src/main/java/de/flux/playground/deckcompare/Deckcompare.java

```diff
@@ -1,21 +1,34 @@
-package de.flux.playground.deckcompare.controller;
+package de.flux.playground.deckcompare;
 
+import java.io.File;
+
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.ResponseBody;
 
 @Controller
-@EnableAutoConfiguration
+@SpringBootApplication
 public class Deckcompare {
 
+    public static String ROOT = "upload-dir";
```

Comment:
You should better make this configurable via `application.properties`. You can inject values from `application.properties` using the `@Value` annotation.